package.json

```diff
@@ -4,7 +4,7 @@
     "email": "info@sussol.net"
   },
   "name": "mSupplyMobile",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
As I said when you made this comment earlier @Chris-Petty, I don't think you can do this with our smart android versioning atm. Can try to make it happen after looking at a couple of higher priorities, but have made v2.0.0 for now